py_mmd_tools/log_util.py

```diff
@@ -0,0 +1,46 @@
+"""
+Tool for converting metadata from MMD format to ISO format using a specific xslt.
+ License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/py-mmd-tools).
```

Comment:
Should be: `License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)`